package.json

```
@@ -7,10 +7,12 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^14.4.3",
+    "node-sass": "^7.0.3",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-redux": "^8.0.5",
     "react-scripts": "5.0.1",
+    "uuid": "^9.0.0",
     "web-vitals": "^2.1.4"
   },
```

Can we define project dependency explicitly ?
```suggestion
  "engines": {
    "node": "18.x",
    "npm": "9.3.x",
    "yarn": "1.22.x"
  },
```